pyproject.toml

```diff
@@ -54,7 +54,8 @@ ipython = "^8.21.0"
 jsonschema = "^4.21.1"
 keyring = "*"
 protobuf = "^4.25.3"
-psycopg2 = "^2.9.9"
+psycopg2 = {markers = "platform_machine == 'aarch64'", version = "^2.9.9"} # https://github.com/canvas-medical/canvas/pull/13880
+psycopg2-binary = {markers = "platform_machine != 'aarch64'", version = "^2.9.9"}
```

Comment:
I had to change this to get things working in my virtual environment. This matches what we have in the `pyproject.toml` file for home-app.